diff -Nur o.sptk-3.5.7.14/examples/themes_test.cpp n.sptk-3.5.7.14/examples/themes_test.cpp
--- o.sptk-3.5.7.14/examples/themes_test.cpp	2008-06-28 07:43:06.000000000 +0200
+++ n.sptk-3.5.7.14/examples/themes_test.cpp	2008-07-13 23:12:22.496669695 +0200
@@ -31,7 +31,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
-#include <fltk-1.1/FL/Fl_Group.H>
+#include <FL/Fl_Group.H>
 #include <iostream>
 
 using namespace std;
diff -Nur o.sptk-3.5.7.14/src/gui/CPngImage.cpp n.sptk-3.5.7.14/src/gui/CPngImage.cpp
--- o.sptk-3.5.7.14/src/gui/CPngImage.cpp	2008-06-28 07:43:06.000000000 +0200
+++ n.sptk-3.5.7.14/src/gui/CPngImage.cpp	2008-07-13 23:13:00.519999307 +0200
@@ -39,7 +39,7 @@
 #include <png.h>
 #ifdef __UNIX_COMPILER__
 #include <unistd.h>
-#include <fltk-1.1/FL/Fl_Image.H>
+#include <FL/Fl_Image.H>
 #endif
 
 using namespace sptk;
diff -Nur o.sptk-3.5.7.14/src/themes/CThemes.cpp n.sptk-3.5.7.14/src/themes/CThemes.cpp
--- o.sptk-3.5.7.14/src/themes/CThemes.cpp	2008-06-28 09:16:50.000000000 +0200
+++ n.sptk-3.5.7.14/src/themes/CThemes.cpp	2008-07-13 23:13:16.473747612 +0200
@@ -47,7 +47,7 @@
 
 #include <iostream>
 #include <sstream>
-#include <fltk-1.1/FL/Fl_Image.H>
+#include <FL/Fl_Image.H>
 
 #include "ThemeUtils.h"
 
